lms/djangoapps/ccx/migrations/0002_auto__del_ccxfuturemembership__del_ccxmembership.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Deleting model 'CcxFutureMembership'
+        db.delete_table('ccx_ccxfuturemembership')
+
+        from ccx_keys.locator import CCXLocator
+        for membership in orm.CcxMembership.objects.all():
+            user = orm.User.objects.get(pk=membership.student)
+            ccx = orm.CustomCourseForEdx.objects.get(pk=membership.ccx)
+            course_key = CCXLocator.from_course_locator(ccx.course_id, ccx.id)
+            enrollment = orm.CourseEnrollment.objects.create(
+                    user=user,
+                    course_id=course_key,
+            )
+            enrollment.mode = "honor"
+            enrollment.is_active = membership.active
+            enrollment.save()
+        # Deleting model 'CcxMembership'
+        db.delete_table('ccx_ccxmembership')
+
+
+    def backwards(self, orm):
+        # Adding model 'CcxFutureMembership'
+        db.create_table('ccx_ccxfuturemembership', (
+            ('id', self.gf('django.db.models.fields.AutoField')(primary_key=True)),
+            ('email', self.gf('django.db.models.fields.CharField')(max_length=255)),
+            ('ccx', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['ccx.CustomCourseForEdX'])),
+            ('auto_enroll', self.gf('django.db.models.fields.BooleanField')(default=False)),
+        ))
+        db.send_create_signal('ccx', ['CcxFutureMembership'])
+
+        # Adding model 'CcxMembership'
+        db.create_table('ccx_ccxmembership', (
+            ('active', self.gf('django.db.models.fields.BooleanField')(default=False)),
+            ('ccx', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['ccx.CustomCourseForEdX'])),
+            ('student', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['auth.User'])),
+            ('id', self.gf('django.db.models.fields.AutoField')(primary_key=True)),
+        ))
+        db.send_create_signal('ccx', ['CcxMembership'])
```

Comment:
Shouldn't the backwards migration also take all of the CourseEnrollments for courses with a CCX course key, copy them to the old CcxMembership, and remove them from CourseEnrollment?
If not, how will the platform handle enrollments for CCX course keys in the CourseEnrollments table if a rollback is needed? E.g. will the student dashboard 500, or perhaps those enrollments won't show up?